docker/test/performance-comparison/report.py

```diff
@@ -206,10 +205,10 @@ def print_changes():
             if int(row[4]):
                 if float(row[2]) < 0.:
                     faster_queries += 1
-                    attrs[2] = 'style="background: #adbdff"'
+                    attrs[2] = 'style="background: #00ff00"'
```

Comment:
Red-green color contrast is not accessible for people with deuteranomaly, which is the most prevalent form of color blindness. 
Using any kind color contrast as the only way to convey information is also not very good for accessibility, but that's another story.